src/components/LicensePicker/style.css

```diff
@@ -0,0 +1,25 @@
+.spdx-picker {
+  font-family: 'Montserrat', sans-serif !important;
+}
+
+.spdx-picker-header-title {
```

Comment:
I'm always torn on this style naming and reuse. On the one hand, this name is awesomely specific and makes it very clear that it drives the style of the picker header. OTOH, lots of things need to be centered and there are many ways of centering. Having that technique defined in one place and shared is awesome. Thoughts?

Reply:
It depends on the specific behavior. In this case, since the style applies only a center, we can define some general classes and use that ones.
But given this could be a standalone component, we should leave also a custom class in order to let the user to customize it as he wants.